pubsub/gcp/pubsub/metadata.go

```diff
@@ -16,4 +16,6 @@ type metadata struct {
 	ClientCertURL           string
 	DisableEntityManagement bool
 	EnableMessageOrdering   bool
+	MaxReconnectionAttempts int `json:"maxReconnectionAttempts"`
+	ConnectionRecoveryInSec int `json:"connectionRecoveryInSec"`
```

Comment:
Are these json tags useful?

Reply:
good question - I put them in because I noticed they are defined in the Azure Service Bus implementation, but happy to remove if not useful

Reply:
There is no usage, you can remove them.

Reply:
done - removed json tags